### PR TITLE
Add uni02beta-upstream trigger job

### PR DESCRIPTION
**Pull request by rlandy**
Trigger the uni02beta-upstream job on changes
to dt/uni02beta files.

Depends-On: https://github.com/openstack-k8s-operators/ci-framework/pull/2435

### CODE REVIEW

File: zuul.d/trigger-jobs.yaml [hunk 1]
```diff
@@ -12,3 +12,14 @@
...
+- job:
+    name: architecture-downstream-uni02beta-upstream-trigger-job
+    parent: downstream-uni02beta-upstream-trigger-job
+    description: |
+      This job will trigger a uni02beta-upstream job downstream.
+      We can invoke this job by adding following comment
+      `\trigger github-experimental` in the github pull request comment.
+      These jobs runs in github-experimental-trigger pipeline.
+    files:
+      - ^dt/uni02beta
```
Comment (pablintino) [inline zuul.d/trigger-jobs.yaml:25]: Care to add these patterns?
```yaml
- ^lib
-  ^automation/vars/uni02beta.yaml
```

Comment (rlandy) [reply zuul.d/trigger-jobs.yaml:25]: done

**Comment by rlandy** [comment]
Testrpoject job: architecture-downstream-uni02beta-upstream-trigger-job https://softwarefactory-project.io/zuul/t/rdoproject.org/build/8b92669bb1384137af31e8a90742abdb : SUCCESS in 1m 19s

Downstream triggered:
[reserve-node-uni02beta-no-overlay]
[uni02beta-no-overlay-upstream-rhel9-rhoso18.0]

**Comment by abays** [comment]
The downstream jobs triggered and passed, so this seems good to me

**Review by abays — APPROVED** [review]
/lgtm
/approve

**Pull request merged**